app/models/notice_submission_finalizer.rb

```diff
@@ -14,10 +14,20 @@ def finalize
     parameters[:submission_id] = notice.id
     notice.works.delete(NoticeSubmissionInitializer::PLACEHOLDER_WORKS)
     notice.update_attributes(parameters)
+    DomainCount.update_count(give_domains_from_urls(notice.infringing_urls))
   end
 
   private
 
   attr_reader :parameters
   attr_accessor :notice
+
+  def give_domains_from_urls(infringing_urls_list)
+    infringing_urls_list.map{ |x| extract_domain(x.url) }.uniq
```

Comment:
I'm going to drop a `.compact` on this to handle the case where `extract_domain` returns `nil` (which it will do if the URL is malformed).